.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
     esac
   - php --ri curl || true
   - ifconfig || true
-  - composer --no-interaction install
+  - composer --no-interaction install --ignore-platform-reqs
```

Comment:
This should be reverted once the dev dependencies are compatible with PHP 8.0.

Reply:
Instead of `--ignore-platgorm-reqs` we could tell Composer to assume that the current PHP version is 7.4 instead of 8 (see https://github.com/mlocati/ip-lib/blob/1.12.0/.travis.yml#L101 for an example)

Reply:
Done in b21c12305cbd114b35b436486d83dabb152a6fc1